src/platforms/common/configuration/options.mdx

```diff
@@ -312,14 +312,22 @@ Configures a separate proxy for outgoing HTTPS requests. This value might not be
 
 </ConfigKey>
 
-<ConfigKey name="cache-directory-path" supported={["dotnet", "android"]}>
+<ConfigKey name="cache-directory-path" supported={["dotnet", "android"]} notSupported={["unity"]}>
```

Comment:
This seems very wrong. But all of `Transport Options` is inside a PlatformSection. So I can't just switch for individual lines. So how else would I go about it?

Reply:
I agree, duplication means the content becomes wrong in at least one place. Rather than duplicate, I'd suggest updating the `Default` content (I'll edit that below)

Reply:
If that doesn't make sense, lmk, and I'll play with it a bit more.